com.reprezen.swagedit.dependencies/build.properties

```diff
@@ -1,20 +1,2 @@
 bin.includes = META-INF/,\
-               lib/,\
-               lib/com.fasterxml.jackson.core.jackson-annotations-2.9.0.jar,\
-               lib/com.fasterxml.jackson.core.jackson-core-2.9.8.jar,\
-               lib/com.fasterxml.jackson.core.jackson-databind-2.9.8.jar,\
-               lib/com.fasterxml.jackson.dataformat.jackson-dataformat-yaml-2.9.8.jar,\
-               lib/com.github.fge.btf-1.2.jar,\
-               lib/com.github.fge.msg-simple-1.1.jar,\
-               lib/com.github.fge.uri-template-0.9.jar,\
-               lib/com.github.java-json-tools.jackson-coreutils-1.9.jar,\
-               lib/com.github.java-json-tools.json-schema-core-1.2.10.jar,\
-               lib/com.github.java-json-tools.json-schema-validator-2.2.10.jar,\
-               lib/com.google.code.findbugs.jsr305-3.0.1.jar,\
-               lib/com.googlecode.libphonenumber.libphonenumber-8.0.0.jar,\
-               lib/com.reprezen.jsonoverlay.jsonoverlay-4.0.4.jar,\
-               lib/com.reprezen.kaizen.openapi-parser-4.0.4.jar,\
-               lib/joda-time.joda-time-2.9.7.jar,\
-               lib/net.sf.jopt-simple.jopt-simple-5.0.3.jar,\
-               lib/org.mozilla.rhino-1.7.7.1.jar,\
-               lib/org.yaml.snakeyaml-1.23.jar
+               lib/
```

Comment:
It looks like we previously listed `/lib` _AND_ then listed individual .jar files under /lib.  Now we only include `/lib`. Why...?